### PR TITLE
AG-3390 - Charts prod updates

### PR DESCRIPTION
**Pull request by taktran**

### CODE REVIEW

File: external/ag-website-shared/src/content/license-features/chartsFeaturesMatrix.json [hunk 1]
```diff
@@ -307,17 +307,17 @@
             },
             {
                 "label": {
-                    "name": "Annotations",
-                    "link": "https://ag-grid.com/charts/react/annotations/"
+                    "name": "Toolbar",
```
Comment (seanlandsman) [inline external/ag-website-shared/src/content/license-features/chartsFeaturesMatrix.json:310]: is annotations no longer an option?

**Review by seanlandsman — APPROVED** [review]
looks fine to me but I think I lack sufficient context to know if this is correct

File: external/ag-website-shared/src/content/license-features/chartsFeaturesMatrix.json (hunk 1 above)
Comment (taktran) [reply external/ag-website-shared/src/content/license-features/chartsFeaturesMatrix.json:310]: The link doesn't seem to be available anymore. Seems like it moved to https://ag-grid.com/charts/react/financial-charts-toolbar/#annotations, but I'll check with David

Comment (taktran) [reply external/ag-website-shared/src/content/license-features/chartsFeaturesMatrix.json:310]: Yep, it's as it should be

**Pull request merged**